js/src/modals/ExportAccount/ExportStore.js

```diff
@@ -0,0 +1,89 @@
+// Copyright 2015-2017 Parity Technologies (UK) Ltd.
```

Comment:
Just rename the file as `exportStore.js` to fit in with the rest of the codebase.